gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -809,17 +851,18 @@ message TripDescriptor {
     // A trip that existed in the schedule but was removed.
     CANCELED = 3;
 
-    // Should not be used - for backwards-compatibility only.
-    REPLACEMENT = 5 [deprecated=true];
+    // A trip that replaces an existing trip in the schedule.
+    REPLACEMENT = 5;
```

Comment:
I think the definition currently implies that, for a given static trip, there can only be one `REPLACEMENT` trip—do stakeholders think this is to be sufficient for their use cases?

I can envision a situation where a scheduled rail trip is broken into multiple sections (eg, due to a suspension of the middle of the line) operated by multiple vehicles. We could handle that by marking one of the new trip as `REPLACEMENT` and the others as `ADDED`, but it doesn't achieve the same outcome.

Reply:
I don't think it is possible to do that given that trip IDs are unique on a service day apart from frequency-based trips, for which the same trip runs multiple times per day. 

If such suspension happens, in effect there will be two trips, hence two trip IDs must be needed.